az_quickstart.profile

```diff
@@ -81,3 +81,12 @@ function az_quickstart_update_9207() {
   $module_list = ['az_security'];
   \Drupal::service('module_installer')->install($module_list);
 }
+
+/**
+ * Ensure Pantheon sites have the advanced page cache module installed.
+ */
+function az_quickstart_update_9208() {
+  if (defined('PANTHEON_ENVIRONMENT')) {
+    \Drupal::service('module_installer')->install(['pantheon_advanced_page_cache']);
```

Comment:
if the module is already installed the `module_installer` service will do nothing, which means this is fine the way it is.

Reply:
`module_installer` does throw a `Drupal\Core\Extension\MissingDependencyException` if the module is missing and then the update would fail. If it's possible someone is using Quickstart on Pantheon, but doesn't have this module, we may want to wrap this in a `try... catch`. I don't know how likely that is -- Pantheon Build Tools-generated sites do include the module but don't enable it, just like the upstream.

Also, I don't know if we want this is in all Pantheon environments including multidev, Dev, and Test. I don't know that we don't, either. I guess that would only matter if sites are using the other environments and have the ability to vary config between them.

Reply:
> `module_installer` does throw a `Drupal\Core\Extension\MissingDependencyException` if the module is missing and then the update would fail. If it's possible someone is using Quickstart on Pantheon, but doesn't have this module, we may want to wrap this in a `try... catch`. I don't know how likely that is -- Pantheon Build Tools-generated sites at least do include the module but don't enable it, just like the upstream.
> 
> Also, I don't know if we want this is in all Pantheon environments including multidev, Dev, and Test. I don't know that we don't, either. I guess that would only matter if sites are using the other environments and have the ability to vary config between them.

I can add a `try...catch`.  That seems like a reasonable change.

Re: the different Pantheon environments:  we haven't run into any trouble having the Pantheon page cache module installed in dev/multidev on D7 sites, I think especially because we use a `settings.php` config override to disable core's anonymous caching setting(s) in all environments besides test and live.

This related upstream PR is for adding the same config overrides to our Quickstart 2 Pantheon upstream: az-digital/az-quickstart#53.  I'd probably recommend anyone using Quickstart 2 on Pantheon without our upstream add something similar to their sites.

Reply:
Got it, thanks!